src/Components/ShareProfile.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import GetIcons from './Icons/GetIcons'
+import PropTypes from 'prop-types'
+import './ShareProfile.css'
+
+export default function ShareProfile({ username }) {
+  const CopyLink = () => {
+    navigator.clipboard.writeText(`http://linkfree.eddiehub.org/${username}`)
+    alert('Link copied to clipboard.')
+  }
+
+  return (
+    <div className="profile__container">
+      <a
```

Comment:
Just a suggestion: Can we make `Icon-Link` as a seperate component with the corresponding props? Otherwise we are having the same code-block multiple times, with just different blocks?

Reply:
aha, sure. On it!

Reply:
Componentized it!